internal/b/blueprints.go

```diff
@@ -53,6 +54,8 @@ type User struct {
 	DisplayName string
 	AvatarURL   string
 	AccountData AccountData
+	DeviceId    *string
+	E2E         bool
```

Comment:
Please document what this will do if `true`. I feel it would probably be more suitable to have `OneTimeKeys int`, and set up device/OTKs if `> 0`.